bool ctrl_sig
bool pwr_sig
---
string status